<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userpage</class>
 <widget class="QMainWindow" name="userpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1304</width>
    <height>847</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>JetBrainsMono Nerd Font</family>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>userpage</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="header_search">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>1021</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLineEdit" name="search_in">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>691</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>111</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>JetBrainsMono Nerd Font</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Shelfwise</string>
     </property>
    </widget>
    <widget class="QPushButton" name="search_but">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>30</y>
       <width>87</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(32, 148, 243);
color:rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
